### ISSUE TITLE
I can't see any private trackers in trackers list

### ISSUE THREAD

**Issue by yassinei** [elgatito/script.elementum.burst#75]

**Comment by elgatito** [comment]
You give too much information.

**Comment by yassinei** [comment]
Haha sorry. Well version 0.0.61

**Comment by elgatito** [comment]
What, where, when, log?!

**Comment by yassinei** [comment]
OK in Burst settings, Providers section, I can only see Public trackers, and the first one mentioned is MagnetDL, and the last is YourBitTorrent, and that's all.

**Comment by elgatito** [comment]
Kodi18?

**Comment by yassinei** [comment]
Yes exactly! Sorry for not mentioning that before. Is it a known issue?

**Comment by elgatito** [comment]
Yes. Kodi 18 is breaking options, that start with a number. There is knows way to overcome it, I don't have Kodi18 to test.

**Comment by elgatito** [comment]
Please, try 0.0.28

**Comment by yassinei** [comment]
Now I can see 1337x on the top, but still no private trackers

**Comment by elgatito** [comment]
Looks like Kodi18 has limit on number of settings in one category. Sad enough.
